user-interface/0002-WCON-connect_vega_wallet.md

```diff
@@ -0,0 +1,79 @@
+# Connect Vega wallet & select keys
+
+## Connect wallet
+
+When looking to use Vega via a user interface e.g. Dapp (Decentralized web App), I...
+
+- **should** see a link to get a Vega wallet (in case I don't already have one) (<a name="0002-WCON-036" href="#0002-WCON-036">0002-WCON-036</a>)
+- **should** see a link to connect that opens up connection options (<a name="0002-WCON-001" href="#0002-WCON-001">0002-WCON-001</a>)
```

Comment:
Maybe say 'call to action' or 'button' rather than link so as not to be confused with actual page links

Reply:
To be honest I don't think it matters whether it is a link or button. Both interns of what gets implemented or how these ACs are worded but have updated: 
```suggestion
- **should** see a call to action to connect that opens up connection options (<a name="0002-WCON-001" href="#0002-WCON-001">0002-WCON-001</a>)
```

Reply:
Sorry to clarify, in this case it wil never be a link as it'll exist within the app itself and not a separate page. It is a minor one though 😄